### PR TITLE
DEV-911 update alloance when open modal

### PR DESCRIPTION
**Pull request by clarkjoao**

### CODE REVIEW

File: components/create-bounty-modal.tsx [hunk 1]
```diff
@@ -611,13 +611,15 @@ export default function CreateBountyModal() {
   }, [customTokens]);
 
   useEffect(() => {
+    let approved = true
```
Comment (vhcsilva) [inline components/create-bounty-modal.tsx:614]: It should be `false` by default, this way we prevent the user from getting the `exceeds allowance` error if the app fails to update the allowance

File: components/create-bounty-modal.tsx [hunk 2]
```diff
@@ -636,6 +638,10 @@ export default function CreateBountyModal() {
     getTokenInfo(activeNetwork.tokens);
   }, [activeNetwork?.tokens, settings]);
 
+  useEffect(()=>{
+    transactionalERC20.updateAllowanceAndBalance();
```
Comment (vhcsilva) [inline components/create-bounty-modal.tsx:642]:
```suggestion
    transactionalERC20.updateAllowanceAndBalance();
    rewardERC20.updateAllowanceAndBalance();
```

**Review by vhcsilva — CHANGES_REQUESTED** [review]

File: components/create-bounty-modal.tsx (hunk 1 above)
Comment (clarkjoao) [reply components/create-bounty-modal.tsx:614]: yeah, i thought the same, but when the type is foundingBounty the approval must be true, therefore i will improve that.

Comment (moshmage) [reply components/create-bounty-modal.tsx:614]: I'd add that explanation as a comment and I think it does suffice

**Review by moshmage — APPROVED** [review]

**Review by vhcsilva — APPROVED** [review]

**Pull request merged**